styles/components/redirection.css

```diff
@@ -0,0 +1,31 @@
+.redirection {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  justify-items: center;
+  background-color: #332f3c;
+  color: #fff;
+  padding: 16px;
+  grid-column-gap: 12px;
+}
+
+.redirection__heading {
+  grid-column: 1/-1;
+  text-transform: uppercase;
+  font-weight: 600;
+  font-size: 24px;
+  letter-spacing: 0.215em;
+  margin: 6px 0 10px;
+}
+
+.redirection__paragraph {
+  grid-column: 1/-1;
+  margin: 0 0 64px;
+}
+
+.redirection__button--pink {
+  justify-self: right;
+}
+
+.redirection__button--black {
+  justify-self: left;
+}
```

Comment:
Class names and properties don't fit together. Better `redirection__button--left` as color does not matter here. Also: why not using the parent element to justify both?